.gitignore

```diff
@@ -3,6 +3,7 @@
 .buildpath
 .project
 .idea
+.che
```

Comment:
What is `.che`? Does this belong to your local environment's global `.gitignore` instead?